claim-db-worker/components/prismaSchemaEditor/SidebarActions.tsx

```
@@ -0,0 +1,93 @@
+  hasConnectionString,
+  isMounted,
+}: Props) {
+  return (
+    <div className="w-full lg:w-16 h-auto lg:h-auto rounded-lg bg-step flex flex-row lg:flex-col justify-between items-center py-2 px-2 lg:px-0 gap-2 lg:gap-0">
+      <div className="flex flex-row lg:flex-col items-center space-y-0 lg:space-y-1 space-x-2 lg:space-x-0 w-full lg:w-auto justify-between lg:justify-start">
+        <button
+          onClick={onPush}
+          disabled={isPushing || isPulling || !hasConnectionString || !isMounted}
+          className="flex flex-row lg:flex-col items-center justify-center rounded-md text-muted hover:text-white hover:bg-button transition-colors disabled:opacity-50 disabled:cursor-not-allowed p-2 lg:aspect-square border border-subtle lg:border-0 w-full lg:w-auto"
+          title={!hasConnectionString ? "No connection string available" : !isMounted ? "Editor initializing..." : "Push schema to database (prisma db push)"}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)